browsers/nodejs.json

```diff
@@ -48,6 +358,36 @@
           "release_notes": "https://nodejs.org/en/blog/release/v10.0.0/",
           "status": "current"
         },
+        "10.0.0": {
```

Comment:
This version [is already present](#diff-2f2da23b76b41b4b8f74813e5959cfd8R356): https://github.com/mdn/browser-compat-data/blob/c0315c6eb6e6efae076e4efaf8b1451f6a3758f4/browsers/nodejs.json#L356-L360